blocks/vue/registry/default/password-based-auth/vue/registry-item.json

```diff
@@ -7,19 +7,23 @@
   "files": [
     {
       "path": "registry/default/password-based-auth/vue/components/login-form.vue",
-      "type": "registry:component"
+      "type": "registry:component",
+      "target": "components/login-form.vue"
```

Comment:
You don't need target here. If you use a target, it will skip whatever config you've setup in components.json. Not sure if it applies to shadcn-vue though.